server/src/main/java/io/deephaven/server/console/completer/PythonAutoCompleteObserver.java

```diff
@@ -70,128 +70,87 @@ public void onNext(AutoCompleteRequest value) {
-                // in the future we may want to get more interesting info from jedi to pass back to client
-                final List<PyObject> items = result.asList();
-                String completionName = items.get(0).getStringValue();
-                int start = items.get(1).getIntValue();
-                final CompletionItem.Builder item = CompletionItem.newBuilder();
-                final TextEdit.Builder textEdit = item.getTextEditBuilder();
-                textEdit.setText(completionName);
-                final DocumentRange.Builder range = textEdit.getRangeBuilder();
-                item.setStart(start);
-                item.setLabel(completionName);
-                item.setLength(completionName.length());
-                range.getStartBuilder().setLine(pos.getLine()).setCharacter(start);
-                range.getEndBuilder().setLine(pos.getLine()).setCharacter(pos.getCharacter());
-                item.setInsertTextFormat(2);
-                item.setSortText(ChunkerCompleter.sortable(finalItems.size()));
-                finalItems.add(item.build());
-            }
-
-            final long nanosBuiltResponse = System.nanoTime();
 
-            final GetCompletionItemsResponse builtItems = GetCompletionItemsResponse.newBuilder()
-                    .setSuccess(true)
-                    .setRequestId(request.getRequestId())
-                    .addAllItems(finalItems)
-                    .build();
+            AutoCompleteResponse.Builder response = AutoCompleteResponse.newBuilder();
 
-            try {
-                safelyOnNext(responseObserver,
-                        AutoCompleteResponse.newBuilder()
-                                .setCompletionItems(builtItems)
-                                .build());
-            } finally {
-                // let's track how long completions take, as it's known that some
-                // modules like numpy can cause slow completion, and we'll want to know what was causing them
-                final long totalCompletionNanos = nanosBuiltResponse - startNano;
-                final long totalJediNanos = nanosJedi - startNano;
-                final long totalResponseBuildNanos = nanosBuiltResponse - nanosJedi;
-                // only log completions taking more than 100ms
-                if (totalCompletionNanos > HUNDRED_MS_IN_NS && log.isTraceEnabled()) {
-                    log.trace().append("Found ")
-                            .append(finalItems.size())
-                            .append(" jedi completions from doc ")
-                            .append(doc.getVersion())
-                            .append("\tjedi_time=").append(toMillis(totalJediNanos))
-                            .append("\tbuild_response_time=").append(toMillis(totalResponseBuildNanos))
-                            .append("\ttotal_complete_time=").append(toMillis(totalCompletionNanos))
-                            .endl();
+            switch (request.getRequestCase()) {
```

Comment:
I think if we ever see the `default` case (in the way the code is currently structured), we should throw an illegal argument exception.